samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -133,11 +171,26 @@ export const ChatInput: React.FC<ChatInputProps> = (props) => {
                         // void chat.sendTypingStartSignalAsync();
                     }}
                 />
-                <div className={classes.controls}>
+            </div>
+            <div className={classes.controls}>
+                <div className={classes.functional}>
+                    {/* Hidden input for file upload. Only accept .txt files for now. */}
+                    <input
+                        type="file"
+                        ref={documentFileRef}
+                        style={{ display: 'none' }}
+                        accept='.txt'
+                        multiple={false}
```

Comment:
What would it take to support multiple? This could be a follow-up item if its a bigger change.